kasa/cli.py

```diff
@@ -81,22 +82,42 @@ def cli(ctx, host, alias, target, debug, bulb, plug, strip):
 
 
 @cli.command()
-@click.option("--save")
+@click.option("--scrub", is_flag=True)
```

Comment:
`--scrub/--no-scrub` and set `default=True`, I think the normal case  for this should be scrub automatically.

Reply:
That makes sense, should I just make it --no-scrub default=False and just not no_scrub on line 104?

Reply:
However you like, usually it's done by having both of those so one can explicitly state their wish, but in this case it's not really that crucial. Here's the documentation for click's boolean flags showing the example: https://click.palletsprojects.com/en/5.x/options/#boolean-flags